src/components/details/market/MarketDetails.tsx

```diff
@@ -68,7 +70,7 @@ function MarketDetails({ marketData, refresh }: Props) {
           {canBuy && <Buy marketData={marketData} refresh={refresh} />}
           {canBid && !hasBidsFromUser && <Bid marketData={marketData} refresh={refresh} />}
           {canBid && hasBidsFromUser && <UpdateBid bids={bidsFromUser} refresh={refresh} />}
```

Comment:
Probably should use `canCancelBid` for this check as well.

Reply:
Use `canCancelBid` in place of what is currently there or use it in addition?